internal/rpc/rpcserver/server.go

```diff
@@ -2019,12 +2041,14 @@ func (s *walletServer) ValidateAddress(ctx context.Context, req *pb.ValidateAddr
 		_, script := ka.PaymentScript()
 		result.PayToAddrScript = script
```

Comment:
this field is known after just decoding the address, without looking up the address in the wallet, so we should be setting these fields of the result above